extensions/wikia/Recirculation/js/recirculation.js

```diff
@@ -1,12 +1,13 @@
 /*global define*/
 define('ext.wikia.recirculation.recirculation', [
 	'jquery',
+	'wikia.window',
 	'wikia.abTest',
 	'wikia.tracker',
 	'wikia.nirvana',
 	'videosmodule.controllers.rail',
 	'ext.wikia.adEngine.taboolaHelper'
-], function ($, abTest, tracker, nirvana, videosModule, taboolaHelper) {
+], function ($, w, abTest, tracker, nirvana, videosModule, taboolaHelper) {
```

Comment:
These parameters are the aliases for the dependencies defined in the array above. We don't really use this module anywhere else, so by adding `wikia.window` to the array at the beginning we can add `w` at the beginning as well.